glossary.md

```diff
@@ -1,42 +1,56 @@
-  * A [distributed, open-source document store](https://www.elastic.co/what-is/elasticsearch) used to store various types of data. HRI uses ElasticSearch as its primary data store for metadata about batches. 
-  
-#### Event Streams: 
-  * The IBM Cloud [Event Streams service](https://www.ibm.com/cloud/event-streams) is a "Managed" service instance of [Apache Kafka](https://kafka.apache.org) customized to work with the IBM Cloud.
-  
-#### HRI: 
-  * The Health Record Ingestion service provides a "front door" for Data Integrators to send data into the IBM Cloud, thereby allowing that data to be used by authorized applications that are part of that Cloud account. 
-
-#### IBM Cloud Functions: 
-  * A "Serverless", auto-scaling RESTful application service framework in the IBM Cloud. See this [Cloud Docs](https://cloud.ibm.com/docs/openwhisk?topic=openwhisk-getting-started) page for more details. 
-  
-#### Management API: 
-  * HRI is designed as a RESTful service layer with an [OpenAPI 3.0](http://spec.openapis.org/oas/v3.0.3) compliant REST API. The "Management API" is the external API layer for accessing all public HRI operations that handle Batch, Streams, and Tenant management. 
-
-#### Multitenancy: 
-  * In the context of HRI, multitenancy is a software architecture pattern that allows multiple users or customers to use a single instance of a specific service, sharing computing resources across those customers. For reference, [see this link](https://www.ibm.com/cloud/learn/multi-tenant).  
-  
-#### PHI: 
-  * Protected Health Information is a term defined by the HIPAA Law and Privacy Rule which provides that "covered entities" must protect certain sensitive personal information of patients and that patients have certain rights to that information. See [this page for HIPAA defintion](https://www.hhs.gov/answers/hipaa/what-is-phi/index.html) and [this page](https://www.hhs.gov/hipaa/for-professionals/privacy/laws-regulations/index.html) for more in-depth info on HIPAA Privacy Rule and Protected Health Information.
-
-#### Stream: 
-  * An HRI Stream represents the entire flow through the HRI for a given tenant. A Stream always has two kafka topics associated with it: an *.in* topic and a *.notification* topic.
-
-#### Tenant: 
-  * A tenant represents one customer or organization that is a "user" of HRI, on whose behalf a set of Health data is being processed. A tenant can be internal or external to the organization deploying HRI (e.g. IBM Watson Health). All data is isolated between tenants. 
\ No newline at end of file
+# Glossary
+
+Learn these key terms and better understand the Health Record Ingestion service.
+
+#### batch
+
+A collection of Health Data records that must be processed **together** for that dataset to be ingested correctly into an IBM&reg; Cloud instance. Processing partial data causes problems. Similarly, if a processing error occurs with part of the data, the entire batch might need to be rejected.
```

Comment:
I think indenting these helps make them easier to read. The '* ' provided that or we could try a block quote, '|'